src/components/notifications/SMSPartner.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="mb-3">
+        <label for="smspartner-key" class="form-label">{{ $t("smspartnerApikey") }}</label>
+        <HiddenInput id="smspartner-key" v-model="$parent.notification.smspartnerApikey" :required="true" autocomplete="new-password"></HiddenInput>
+        <div class="form-text">
+            {{ $t("smspartnerApiurl") }}
+            <a href="https://my.smspartner.fr/dashboard/api" target="_blank">https://my.smspartner.fr/dashboard/api</a>
+        </div>
+    </div>
+    <div class="mb-3">
+        <label for="smspartner-phone-number" class="form-label">{{ $t("smspartnerPhoneNumber") }}</label>
```

Comment:
please extract the part from `smspartnerPhoneNumber` which is a helptext as a helptext ^^
Maybe reword a little to `multiple numbers must be separated by ","`.

> [!TIP]
> You can use https://vue-i18n.intlify.dev/guide/advanced/component.html to use `<code>,</code>` in the helptext

Reply:
Done ! Thank you :)